components/empty-state/empty-state-styles.js

```diff
@@ -30,16 +36,10 @@ export const emptyStateSimpleStyles = css`
 		padding: 1.2rem 1.5rem;
 	}
 
-	:host([dir="rtl"]) .d2l-empty-state-description {
-		padding-left: 0.5rem;
-		padding-right: 0;
-	}
-
 	.d2l-empty-state-description {
 		display: inline;
-		padding-right: 0.5rem;
+		margin: 0;
```

Comment:
I played around with this a bit. I think in order to get the height right when there's just the description, you also need `line-height: 28px`. I believe you also want to only have the `column-gap` when there's a description, i.e., `:host([description]) .empty-state-container`.

Also, looking at the `normal-link-rtl` (and other link rtl vdiffs) I'm not totally sure what's going on - they now look consistent with the button vdiffs in rtl but I don't know if what's there before was actually considered correct.